docs/classic/backgammon.md

```diff
@@ -92,9 +92,9 @@ The location on the board can be found from the location ID, which is either the
 |>25 | Black's bear off location|
 
 #### Agent Order
-The game starts with rolling two dice until their values are different. If the first roll is larger, then the first agent is assigned the color white. Otherwise, the first agent is assigned the color black.
+The game starts by rolling two dice until their values are different. If the first roll is larger, then the first agent is assigned the color white. Otherwise, the first agent is assigned the color black.
```

Comment:
It seems like it was better before?

Reply:
Yeah, reading it again I think you might be right